crates/attestation/src/snp/mod.rs

```diff
@@ -245,6 +254,9 @@ impl Snp {
 
             if let Some(signer) = signer {
                 if signer == &vcek.tbs_certificate {
+                    //let mut path = path;
+                    //path.push(vcek.clone());
+                    //path.check_crl(&certs.crl)?;
```

Comment:
The CRL check is commented out?

Reply:
Yes, we're not doing CRL checking now since we need to get a release out ASAP, since the communication between Enarx and Steward will be broken otherwise.